sopel/trigger.py

```diff
@@ -236,11 +236,11 @@ class Trigger(unicode):
     time = property(lambda self: self._pretrigger.time)
     """When the message was received.
 
-    :type: :class:`~datetime.datetime` object
+    :type: :class:`~datetime.datetime` object (without a timezone)
```

Comment:
```suggestion
    :type: naïve :class:`~datetime.datetime` object (no timezone)
```

Take or leave; just an idea.